quasar-cli-vue3-webpack-javascript/e2e/CheckAppExposes.cy.ts

```diff
@@ -0,0 +1,302 @@
+                text: String(property.appExposesComponentsButton)
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.appExposesCounter,
+                text: '0'
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.divElement,
+                text: String(property.appListName)
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.divElement,
+                text: String(property.appListDiv)
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 5
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.appExposesCloseName,
+                index: 1
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 4
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.appExposesCloseName,
+                index: 1
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 3
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.appExposesCloseName,
+                index: 1
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 2
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.appExposesCloseName,
+                index: 1
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 1
+            })
+            basePage.openLocalhost(3001)
+            basePage.clickElementWithText({
+                selector: baseSelectors.linkTag,
+                text: String(property.appExposesComponentsButton)
+            })
+            basePage.checkElementQuantity({
+                selector: baseSelectors.appExposesNames,
+                quantity: 5
+            })
+        })
+        it (`Check App General Elements`, () => {
+            basePage.skipTestByCondition(property.host === 3001)
+            basePage.clickElementWithText({
+                selector: baseSelectors.divElement,
+                text: String(property.appGeneralName)
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h4,
+                text: String(property.appGeneralBanner)
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.button
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.button
+            })
```

Comment:
Same) Why second click?

Reply:
same reason) "to open the left side menu and reach the Components button, cause when I click for the first time -> the Left side menu is hiding, also for check this Hamburger button"